nopolinfo #0
location: FailureLocation{className='redis.clients.jedis.tests.ConnectionTest', failingMethods=[redis.clients.jedis.tests.ConnectionTest#getErrorAfterConnectionReset], erroringMethods=[redis.clients.jedis.tests.ConnectionTest#checkCloseable, redis.clients.jedis.tests.ConnectionTest#connectIfNotConnectedWhenSettingTimeoutInfinite], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='org.junit.ComparisonFailure', failureDetail='expected:<[ERR Protocol error: invalid multibulk length]> but was:<[Failed connecting to host localhost:6379]>', isError=false}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=1, nbErrors=2}
status: NOPATCH
dateEnd: Mon Oct 23 18:13:28 CEST 2017
allocatedtime: 120minutes 
passingTime: 5minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/291610330/src/main/java], projectClasspath='[Ljava.net.URL;@148ab138', projectTests=[redis.clients.jedis.tests.ConnectionTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 31
nbAngelicValues: 23
ignoreStatus: IGNORE_ERRORING
----------



nopolinfo #1
location: FailureLocation{className='redis.clients.jedis.tests.ConnectionTest', failingMethods=[redis.clients.jedis.tests.ConnectionTest#getErrorAfterConnectionReset], erroringMethods=[redis.clients.jedis.tests.ConnectionTest#checkCloseable, redis.clients.jedis.tests.ConnectionTest#connectIfNotConnectedWhenSettingTimeoutInfinite], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='org.junit.ComparisonFailure', failureDetail='expected:<[ERR Protocol error: invalid multibulk length]> but was:<[Failed connecting to host localhost:6379]>', isError=false}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=1, nbErrors=2}
status: NOPATCH
dateEnd: Mon Oct 23 18:17:51 CEST 2017
allocatedtime: 117minutes 
passingTime: 4minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/291610330/src/main/java], projectClasspath='[Ljava.net.URL;@4af606e9', projectTests=[redis.clients.jedis.tests.ConnectionTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 31
nbAngelicValues: 23
ignoreStatus: IGNORE_FAILING
----------



nopolinfo #2
location: FailureLocation{className='redis.clients.jedis.tests.JedisClusterTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.JedisClusterTest#testStableSlotWhenMigratingNodeOrImportingNodeIsNotSpecified, redis.clients.jedis.tests.JedisClusterTest#testClusterFlushSlots, redis.clients.jedis.tests.JedisClusterTest#testClusterKeySlot, redis.clients.jedis.tests.JedisClusterTest#testReturnConnectionOnJedisConnectionException, redis.clients.jedis.tests.JedisClusterTest#testRedisClusterMaxRedirections, redis.clients.jedis.tests.JedisClusterTest#testCloseable, redis.clients.jedis.tests.JedisClusterTest#testLocalhostNodeNotAddedWhen127Present, redis.clients.jedis.tests.JedisClusterTest#testMigrate, redis.clients.jedis.tests.JedisClusterTest#testIfPoolConfigAppliesToClusterPools, redis.clients.jedis.tests.JedisClusterTest#testClusterForgetNode, redis.clients.jedis.tests.JedisClusterTest#testCalculateConnectionPerSlot, redis.clients.jedis.tests.JedisClusterTest#testMovedExceptionParameters, redis.clients.jedis.tests.JedisClusterTest#testRecalculateSlotsWhenMoved, redis.clients.jedis.tests.JedisClusterTest#testThrowAskException, redis.clients.jedis.tests.JedisClusterTest#testAskResponse, redis.clients.jedis.tests.JedisClusterTest#testReadonly, redis.clients.jedis.tests.JedisClusterTest#testRedisHashtag, redis.clients.jedis.tests.JedisClusterTest#testJedisClusterTimeout, redis.clients.jedis.tests.JedisClusterTest#testInvalidStartNodeNotAdded, redis.clients.jedis.tests.JedisClusterTest#testJedisClusterRunsWithMultithreaded, redis.clients.jedis.tests.JedisClusterTest#redis.clients.jedis.tests.JedisClusterTest, redis.clients.jedis.tests.JedisClusterTest#testMigrateToNewNode, redis.clients.jedis.tests.JedisClusterTest#testDiscoverNodesAutomatically, redis.clients.jedis.tests.JedisClusterTest#testThrowMovedException, redis.clients.jedis.tests.JedisClusterTest#testClusterCountKeysInSlot, redis.clients.jedis.tests.JedisClusterTest#testReturnConnectionOnRedirection, redis.clients.jedis.tests.JedisClusterTest#testSetClientName], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}], nbFailures=0, nbErrors=53}
status: NOPATCH
dateEnd: Mon Oct 23 18:28:02 CEST 2017
allocatedtime: 115minutes 
passingTime: 10minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/291610330/src/main/java], projectClasspath='[Ljava.net.URL;@2b5ca574', projectTests=[redis.clients.jedis.tests.JedisClusterTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 51
nbAngelicValues: 35
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #3
location: FailureLocation{className='redis.clients.jedis.tests.commands.SortedSetCommandsTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.commands.SortedSetCommandsTest#zrange, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zscanMatch, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zrevrange, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zcard, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zrank, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zintertoreParams, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zrangeByLexBinary, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zunionstore, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zrevrangebyscore, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zlexcountBinary, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zrevrangeByLex, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zincrbyWithParams, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zcount, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zrangebyscoreWithScores, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zremrangeByLex, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zremrangeByScore, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zrangebyscore, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zadd, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zaddWithParams, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zscanCount, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zremrangeByRank, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zunionstoreParams, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zrangeWithScores, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zrevrank, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zinterstore, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zrevrangebyscoreWithScores, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zrem, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zrevrangeWithScores, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zincrby, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zlexcount, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zrangeByLex, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zremrangeByLexBinary, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zscan, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zrevrangeByLexBinary, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zscore], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=35}
status: NOPATCH
dateEnd: Mon Oct 23 18:34:09 CEST 2017
allocatedtime: 110minutes 
passingTime: 6minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/291610330/src/main/java], projectClasspath='[Ljava.net.URL;@18ada4e1', projectTests=[redis.clients.jedis.tests.commands.SortedSetCommandsTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 32
nbAngelicValues: 26
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #4
location: FailureLocation{className='redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#ttl, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#renameOldAndNewAreTheSame, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#swapDB, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#pexpireAt, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#persist, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#expireAt, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#scanMatch, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#existsMany, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#dbSize, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#rename, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#expire, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#psetex, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#move, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#dumpAndRestore, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#getDB, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#flushAll, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#exists, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#randomKey, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#scanIsCompleteIteration, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#select, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#setNxExAndGet, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#echo, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#pingWithMessage, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#del, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#pexpire, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#scanCount, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#pttl, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#flushDB, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#scan, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#type, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#touch, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#renamenx, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#keys, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#ping], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=34}
status: NOPATCH
dateEnd: Mon Oct 23 18:40:16 CEST 2017
allocatedtime: 107minutes 
passingTime: 6minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/291610330/src/main/java], projectClasspath='[Ljava.net.URL;@7beb2bc2', projectTests=[redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 32
nbAngelicValues: 26
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #5
location: FailureLocation{className='redis.clients.jedis.tests.PipeliningTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.PipeliningTest#testEval, redis.clients.jedis.tests.PipeliningTest#pipelineBinarySafeHashCommands, redis.clients.jedis.tests.PipeliningTest#pipelineResponseWithinPipeline, redis.clients.jedis.tests.PipeliningTest#testPipelinedTransactionResponse, redis.clients.jedis.tests.PipeliningTest#testReuseJedisWhenPipelineIsEmpty, redis.clients.jedis.tests.PipeliningTest#testEvalshaKeyAndArg, redis.clients.jedis.tests.PipeliningTest#pipelineResponseWithoutData, redis.clients.jedis.tests.PipeliningTest#testEvalNestedLists, redis.clients.jedis.tests.PipeliningTest#testCloseableWithMulti, redis.clients.jedis.tests.PipeliningTest#pipelineExecShoudThrowJedisDataExceptionWhenNotInMulti, redis.clients.jedis.tests.PipeliningTest#piplineWithError, redis.clients.jedis.tests.PipeliningTest#pipelineSelect, redis.clients.jedis.tests.PipeliningTest#testEvalshaKeyAndArgWithBinary, redis.clients.jedis.tests.PipeliningTest#testDiscardInPipeline, redis.clients.jedis.tests.PipeliningTest#pipelineWithPubSub, redis.clients.jedis.tests.PipeliningTest#pipeline, redis.clients.jedis.tests.PipeliningTest#testEvalsha, redis.clients.jedis.tests.PipeliningTest#canRetrieveUnsetKey, redis.clients.jedis.tests.PipeliningTest#testEvalKeyAndArg, redis.clients.jedis.tests.PipeliningTest#multiWithMassiveRequests, redis.clients.jedis.tests.PipeliningTest#pipelineResponse, redis.clients.jedis.tests.PipeliningTest#multiWithSync, redis.clients.jedis.tests.PipeliningTest#testEvalNestedListsWithBinary, redis.clients.jedis.tests.PipeliningTest#testJedisThowExceptionWhenInPipeline, redis.clients.jedis.tests.PipeliningTest#testSyncWithNoCommandQueued, redis.clients.jedis.tests.PipeliningTest#testResetStateWhenInPipeline, redis.clients.jedis.tests.PipeliningTest#testCloseable, redis.clients.jedis.tests.PipeliningTest#testEvalKeyAndArgWithBinary, redis.clients.jedis.tests.PipeliningTest#testEvalWithBinary, redis.clients.jedis.tests.PipeliningTest#pipelineMultiShoudThrowJedisDataExceptionWhenAlreadyInMulti, redis.clients.jedis.tests.PipeliningTest#pipelineResponseWithData, redis.clients.jedis.tests.PipeliningTest#pipelineDiscardShoudThrowJedisDataExceptionWhenNotInMulti, redis.clients.jedis.tests.PipeliningTest#multi], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=33}
status: NOPATCH
dateEnd: Mon Oct 23 18:45:33 CEST 2017
allocatedtime: 104minutes 
passingTime: 5minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/291610330/src/main/java], projectClasspath='[Ljava.net.URL;@7324d83', projectTests=[redis.clients.jedis.tests.PipeliningTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 28
nbAngelicValues: 21
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #6
location: FailureLocation{className='redis.clients.jedis.tests.commands.ClusterCommandsTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.commands.ClusterCommandsTest#redis.clients.jedis.tests.commands.ClusterCommandsTest, redis.clients.jedis.tests.commands.ClusterCommandsTest#clusterAddSlots, redis.clients.jedis.tests.commands.ClusterCommandsTest#clusterInfo, redis.clients.jedis.tests.commands.ClusterCommandsTest#clusterSetSlotNode, redis.clients.jedis.tests.commands.ClusterCommandsTest#clusterSetSlotImporting, redis.clients.jedis.tests.commands.ClusterCommandsTest#testClusterHardReset, redis.clients.jedis.tests.commands.ClusterCommandsTest#clusterGetKeysInSlot, redis.clients.jedis.tests.commands.ClusterCommandsTest#clusterSlots, redis.clients.jedis.tests.commands.ClusterCommandsTest#testClusterSoftReset, redis.clients.jedis.tests.commands.ClusterCommandsTest#clusterDelSlots, redis.clients.jedis.tests.commands.ClusterCommandsTest#clusterMeet, redis.clients.jedis.tests.commands.ClusterCommandsTest#clusterSetSlotMigrating, redis.clients.jedis.tests.commands.ClusterCommandsTest#clusterNodes], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='java.lang.NullPointerException', failureDetail='null', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='java.lang.NullPointerException', failureDetail='null', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='java.lang.NullPointerException', failureDetail='null', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='java.lang.NullPointerException', failureDetail='null', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='java.lang.NullPointerException', failureDetail='null', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='java.lang.NullPointerException', failureDetail='null', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='java.lang.NullPointerException', failureDetail='null', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='java.lang.NullPointerException', failureDetail='null', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='java.lang.NullPointerException', failureDetail='null', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='java.lang.NullPointerException', failureDetail='null', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='java.lang.NullPointerException', failureDetail='null', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='java.lang.NullPointerException', failureDetail='null', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}], nbFailures=0, nbErrors=25}
status: NOPATCH
dateEnd: Mon Oct 23 18:53:51 CEST 2017
allocatedtime: 102minutes 
passingTime: 8minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/291610330/src/main/java], projectClasspath='[Ljava.net.URL;@5839306e', projectTests=[redis.clients.jedis.tests.commands.ClusterCommandsTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 54
nbAngelicValues: 39
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #7
location: FailureLocation{className='redis.clients.jedis.tests.commands.BinaryValuesCommandsTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#incrWrongValue, redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#setAndGet, redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#mget, redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#msetnx, redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#decrByWrongValue, redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#getSet, redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#incrBy, redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#setIfExistAndGet, redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#setex, redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#setFailIfNotExistAndGet, redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#incrByWrongValue, redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#decrBy, redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#substr, redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#strlen, redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#setAndExpireMillis, redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#setnx, redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#mset, redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#incr, redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#setAndExpire, redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#setIfNotExistAndGet, redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#decr, redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#append, redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#decrWrongValue, redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#setNxExAndGet], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=24}
status: NOPATCH
dateEnd: Mon Oct 23 18:59:28 CEST 2017
allocatedtime: 98minutes 
passingTime: 5minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/291610330/src/main/java], projectClasspath='[Ljava.net.URL;@27b27f3c', projectTests=[redis.clients.jedis.tests.commands.BinaryValuesCommandsTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 32
nbAngelicValues: 26
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #8
location: FailureLocation{className='redis.clients.jedis.tests.commands.StringValuesCommandsTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.commands.StringValuesCommandsTest#setex, redis.clients.jedis.tests.commands.StringValuesCommandsTest#strlen, redis.clients.jedis.tests.commands.StringValuesCommandsTest#decrByWrongValue, redis.clients.jedis.tests.commands.StringValuesCommandsTest#decr, redis.clients.jedis.tests.commands.StringValuesCommandsTest#incrLargeNumbers, redis.clients.jedis.tests.commands.StringValuesCommandsTest#incrWrongValue, redis.clients.jedis.tests.commands.StringValuesCommandsTest#append, redis.clients.jedis.tests.commands.StringValuesCommandsTest#incrByFloatWrongValue, redis.clients.jedis.tests.commands.StringValuesCommandsTest#setAndGet, redis.clients.jedis.tests.commands.StringValuesCommandsTest#incr, redis.clients.jedis.tests.commands.StringValuesCommandsTest#incrByWrongValue, redis.clients.jedis.tests.commands.StringValuesCommandsTest#mset, redis.clients.jedis.tests.commands.StringValuesCommandsTest#msetnx, redis.clients.jedis.tests.commands.StringValuesCommandsTest#incrReallyLargeNumbers, redis.clients.jedis.tests.commands.StringValuesCommandsTest#mget, redis.clients.jedis.tests.commands.StringValuesCommandsTest#incrBy, redis.clients.jedis.tests.commands.StringValuesCommandsTest#psetex, redis.clients.jedis.tests.commands.StringValuesCommandsTest#substr, redis.clients.jedis.tests.commands.StringValuesCommandsTest#decrBy, redis.clients.jedis.tests.commands.StringValuesCommandsTest#getSet, redis.clients.jedis.tests.commands.StringValuesCommandsTest#decrWrongValue, redis.clients.jedis.tests.commands.StringValuesCommandsTest#incrByFloat, redis.clients.jedis.tests.commands.StringValuesCommandsTest#setnx], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=23}
status: NOPATCH
dateEnd: Mon Oct 23 19:05:03 CEST 2017
allocatedtime: 95minutes 
passingTime: 5minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/291610330/src/main/java], projectClasspath='[Ljava.net.URL;@1a61a2c4', projectTests=[redis.clients.jedis.tests.commands.StringValuesCommandsTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 32
nbAngelicValues: 26
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #9
location: FailureLocation{className='redis.clients.jedis.tests.commands.HashesCommandsTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.commands.HashesCommandsTest#hscan, redis.clients.jedis.tests.commands.HashesCommandsTest#hexists, redis.clients.jedis.tests.commands.HashesCommandsTest#testBinaryHstrLen, redis.clients.jedis.tests.commands.HashesCommandsTest#hset, redis.clients.jedis.tests.commands.HashesCommandsTest#testHstrLen_EmptyHash, redis.clients.jedis.tests.commands.HashesCommandsTest#hdel, redis.clients.jedis.tests.commands.HashesCommandsTest#testHstrLen, redis.clients.jedis.tests.commands.HashesCommandsTest#hincrBy, redis.clients.jedis.tests.commands.HashesCommandsTest#hgetAll, redis.clients.jedis.tests.commands.HashesCommandsTest#hvals, redis.clients.jedis.tests.commands.HashesCommandsTest#hkeys, redis.clients.jedis.tests.commands.HashesCommandsTest#hmset, redis.clients.jedis.tests.commands.HashesCommandsTest#hmget, redis.clients.jedis.tests.commands.HashesCommandsTest#hincrByFloat, redis.clients.jedis.tests.commands.HashesCommandsTest#hgetAllPipeline, redis.clients.jedis.tests.commands.HashesCommandsTest#hsetnx, redis.clients.jedis.tests.commands.HashesCommandsTest#hget, redis.clients.jedis.tests.commands.HashesCommandsTest#hlen, redis.clients.jedis.tests.commands.HashesCommandsTest#hscanCount, redis.clients.jedis.tests.commands.HashesCommandsTest#hscanMatch], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=20}
status: NOPATCH
dateEnd: Mon Oct 23 19:11:03 CEST 2017
allocatedtime: 93minutes 
passingTime: 6minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/291610330/src/main/java], projectClasspath='[Ljava.net.URL;@73858ba', projectTests=[redis.clients.jedis.tests.commands.HashesCommandsTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 32
nbAngelicValues: 26
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #10
location: FailureLocation{className='redis.clients.jedis.tests.commands.ClusterScriptingCommandsTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.commands.ClusterScriptingCommandsTest#testJedisClusterException2, redis.clients.jedis.tests.commands.ClusterScriptingCommandsTest#testBinaryScriptKill, redis.clients.jedis.tests.commands.ClusterScriptingCommandsTest#testEvalsha, redis.clients.jedis.tests.commands.ClusterScriptingCommandsTest#testBinaryEval, redis.clients.jedis.tests.commands.ClusterScriptingCommandsTest#testEval2, redis.clients.jedis.tests.commands.ClusterScriptingCommandsTest#testScriptLoadAndScriptExists, redis.clients.jedis.tests.commands.ClusterScriptingCommandsTest#testJedisClusterException, redis.clients.jedis.tests.commands.ClusterScriptingCommandsTest#redis.clients.jedis.tests.commands.ClusterScriptingCommandsTest, redis.clients.jedis.tests.commands.ClusterScriptingCommandsTest#testBinaryScriptFlush, redis.clients.jedis.tests.commands.ClusterScriptingCommandsTest#testBinaryScriptExists], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='java.lang.NullPointerException', failureDetail='null', isError=true}], nbFailures=0, nbErrors=19}
status: NOPATCH
dateEnd: Mon Oct 23 19:19:21 CEST 2017
allocatedtime: 90minutes 
passingTime: 8minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/291610330/src/main/java], projectClasspath='[Ljava.net.URL;@54f10c5c', projectTests=[redis.clients.jedis.tests.commands.ClusterScriptingCommandsTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 62
nbAngelicValues: 39
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #11
location: FailureLocation{className='redis.clients.jedis.tests.JedisPoolTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.JedisPoolTest#checkConnectionWithDefaultPort, redis.clients.jedis.tests.JedisPoolTest#returnResourceShouldResetState, redis.clients.jedis.tests.JedisPoolTest#getNumActiveReturnsTheCorrectNumber, redis.clients.jedis.tests.JedisPoolTest#checkConnections, redis.clients.jedis.tests.JedisPoolTest#testCloseConnectionOnMakeObject, redis.clients.jedis.tests.JedisPoolTest#checkJedisIsReusedWhenReturned, redis.clients.jedis.tests.JedisPoolTest#checkPoolRepairedWhenJedisIsBroken, redis.clients.jedis.tests.JedisPoolTest#customClientName, redis.clients.jedis.tests.JedisPoolTest#checkCloseableConnections, redis.clients.jedis.tests.JedisPoolTest#startWithUrl, redis.clients.jedis.tests.JedisPoolTest#checkResourceIsCloseable, redis.clients.jedis.tests.JedisPoolTest#securePool, redis.clients.jedis.tests.JedisPoolTest#testResetInvalidPassword, redis.clients.jedis.tests.JedisPoolTest#startWithUrlString, redis.clients.jedis.tests.JedisPoolTest#nonDefaultDatabase, redis.clients.jedis.tests.JedisPoolTest#selectDatabaseOnActivation, redis.clients.jedis.tests.JedisPoolTest#testResetValidPassword, redis.clients.jedis.tests.JedisPoolTest#checkPoolOverflow, redis.clients.jedis.tests.JedisPoolTest#testAddObject], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Could not get a resource from the pool', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Could not get a resource from the pool', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Could not get a resource from the pool', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Could not get a resource from the pool', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Could not get a resource from the pool', isError=true}, FailureType{failureName='java.lang.Exception', failureDetail='Unexpected exception, expected<redis.clients.jedis.exceptions.JedisExhaustedPoolException> but was<redis.clients.jedis.exceptions.JedisConnectionException>', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Could not get a resource from the pool', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6380', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Could not get a resource from the pool', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Could not get a resource from the pool', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Could not get a resource from the pool', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisException', failureDetail='Error trying to add idle objects', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6380', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Could not get a resource from the pool', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Could not get a resource from the pool', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Could not get a resource from the pool', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Could not get a resource from the pool', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Could not get a resource from the pool', isError=true}], nbFailures=0, nbErrors=19}
status: NOPATCH
dateEnd: Mon Oct 23 19:47:01 CEST 2017
allocatedtime: 86minutes 
passingTime: 27minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/291610330/src/main/java], projectClasspath='[Ljava.net.URL;@5348db24', projectTests=[redis.clients.jedis.tests.JedisPoolTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 181
nbAngelicValues: 120
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #12
location: FailureLocation{className='redis.clients.jedis.tests.commands.ListCommandsTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.commands.ListCommandsTest#rpop, redis.clients.jedis.tests.commands.ListCommandsTest#rpush, redis.clients.jedis.tests.commands.ListCommandsTest#llenNotOnList, redis.clients.jedis.tests.commands.ListCommandsTest#ltrim, redis.clients.jedis.tests.commands.ListCommandsTest#brpoplpush, redis.clients.jedis.tests.commands.ListCommandsTest#lrange, redis.clients.jedis.tests.commands.ListCommandsTest#lpop, redis.clients.jedis.tests.commands.ListCommandsTest#rpoplpush, redis.clients.jedis.tests.commands.ListCommandsTest#lindex, redis.clients.jedis.tests.commands.ListCommandsTest#llen, redis.clients.jedis.tests.commands.ListCommandsTest#lpush, redis.clients.jedis.tests.commands.ListCommandsTest#lpushx, redis.clients.jedis.tests.commands.ListCommandsTest#lrem, redis.clients.jedis.tests.commands.ListCommandsTest#brpop, redis.clients.jedis.tests.commands.ListCommandsTest#rpushx, redis.clients.jedis.tests.commands.ListCommandsTest#lset, redis.clients.jedis.tests.commands.ListCommandsTest#blpop, redis.clients.jedis.tests.commands.ListCommandsTest#linsert], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=18}
status: NOPATCH
dateEnd: Mon Oct 23 19:52:26 CEST 2017
allocatedtime: 72minutes 
passingTime: 5minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/291610330/src/main/java], projectClasspath='[Ljava.net.URL;@4c159b48', projectTests=[redis.clients.jedis.tests.commands.ListCommandsTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 32
nbAngelicValues: 26
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #13
location: FailureLocation{className='redis.clients.jedis.tests.commands.SetCommandsTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.commands.SetCommandsTest#scard, redis.clients.jedis.tests.commands.SetCommandsTest#sscanCount, redis.clients.jedis.tests.commands.SetCommandsTest#sadd, redis.clients.jedis.tests.commands.SetCommandsTest#smove, redis.clients.jedis.tests.commands.SetCommandsTest#srem, redis.clients.jedis.tests.commands.SetCommandsTest#sscanMatch, redis.clients.jedis.tests.commands.SetCommandsTest#sunionstore, redis.clients.jedis.tests.commands.SetCommandsTest#srandmember, redis.clients.jedis.tests.commands.SetCommandsTest#spop, redis.clients.jedis.tests.commands.SetCommandsTest#sismember, redis.clients.jedis.tests.commands.SetCommandsTest#sinterstore, redis.clients.jedis.tests.commands.SetCommandsTest#sunion, redis.clients.jedis.tests.commands.SetCommandsTest#smembers, redis.clients.jedis.tests.commands.SetCommandsTest#sdiffstore, redis.clients.jedis.tests.commands.SetCommandsTest#sscan, redis.clients.jedis.tests.commands.SetCommandsTest#sdiff, redis.clients.jedis.tests.commands.SetCommandsTest#sinter, redis.clients.jedis.tests.commands.SetCommandsTest#spopWithCount], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=18}
status: NOPATCH
dateEnd: Mon Oct 23 19:57:58 CEST 2017
allocatedtime: 70minutes 
passingTime: 5minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/291610330/src/main/java], projectClasspath='[Ljava.net.URL;@4cd775f3', projectTests=[redis.clients.jedis.tests.commands.SetCommandsTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 32
nbAngelicValues: 26
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #14
location: FailureLocation{className='redis.clients.jedis.tests.commands.ScriptingCommandsTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.commands.ScriptingCommandsTest#evalsha, redis.clients.jedis.tests.commands.ScriptingCommandsTest#evalInt, redis.clients.jedis.tests.commands.ScriptingCommandsTest#scriptLoadBinary, redis.clients.jedis.tests.commands.ScriptingCommandsTest#evalMultiBulkWithBinaryJedis, redis.clients.jedis.tests.commands.ScriptingCommandsTest#evalNestedLists, redis.clients.jedis.tests.commands.ScriptingCommandsTest#evalshaShaNotFound, redis.clients.jedis.tests.commands.ScriptingCommandsTest#scriptExistsBinary, redis.clients.jedis.tests.commands.ScriptingCommandsTest#scriptKill, redis.clients.jedis.tests.commands.ScriptingCommandsTest#scriptExistsWithBrokenConnection, redis.clients.jedis.tests.commands.ScriptingCommandsTest#evalMultiBulk, redis.clients.jedis.tests.commands.ScriptingCommandsTest#evalBulk, redis.clients.jedis.tests.commands.ScriptingCommandsTest#scriptEvalReturnNullValues, redis.clients.jedis.tests.commands.ScriptingCommandsTest#scriptEvalShaReturnNullValues, redis.clients.jedis.tests.commands.ScriptingCommandsTest#scriptLoad, redis.clients.jedis.tests.commands.ScriptingCommandsTest#scriptFlush, redis.clients.jedis.tests.commands.ScriptingCommandsTest#scriptExists, redis.clients.jedis.tests.commands.ScriptingCommandsTest#evalNoArgs], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=17}
status: NOPATCH
dateEnd: Mon Oct 23 20:03:27 CEST 2017
allocatedtime: 67minutes 
passingTime: 5minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/291610330/src/main/java], projectClasspath='[Ljava.net.URL;@1d960349', projectTests=[redis.clients.jedis.tests.commands.ScriptingCommandsTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 32
nbAngelicValues: 26
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #15
location: FailureLocation{className='redis.clients.jedis.tests.commands.TransactionCommandsTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.commands.TransactionCommandsTest#validateWhenInMulti, redis.clients.jedis.tests.commands.TransactionCommandsTest#testCloseable, redis.clients.jedis.tests.commands.TransactionCommandsTest#transactionResponse, redis.clients.jedis.tests.commands.TransactionCommandsTest#discard, redis.clients.jedis.tests.commands.TransactionCommandsTest#testResetStateWithFullyExecutedTransaction, redis.clients.jedis.tests.commands.TransactionCommandsTest#transactionResponseBinary, redis.clients.jedis.tests.commands.TransactionCommandsTest#testResetStateWhenInMulti, redis.clients.jedis.tests.commands.TransactionCommandsTest#transactionResponseWithinPipeline, redis.clients.jedis.tests.commands.TransactionCommandsTest#testResetStateWhenInMultiWithinPipeline, redis.clients.jedis.tests.commands.TransactionCommandsTest#transactionResponseWithError, redis.clients.jedis.tests.commands.TransactionCommandsTest#testResetStateWhenInWatch, redis.clients.jedis.tests.commands.TransactionCommandsTest#multi, redis.clients.jedis.tests.commands.TransactionCommandsTest#unwatch, redis.clients.jedis.tests.commands.TransactionCommandsTest#select, redis.clients.jedis.tests.commands.TransactionCommandsTest#execGetResponse, redis.clients.jedis.tests.commands.TransactionCommandsTest#watch], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=16}
status: NOPATCH
dateEnd: Mon Oct 23 20:08:54 CEST 2017
allocatedtime: 65minutes 
passingTime: 5minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/291610330/src/main/java], projectClasspath='[Ljava.net.URL;@6f4eff69', projectTests=[redis.clients.jedis.tests.commands.TransactionCommandsTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 32
nbAngelicValues: 26
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #16
location: FailureLocation{className='redis.clients.jedis.tests.commands.PublishSubscribeCommandsTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.commands.PublishSubscribeCommandsTest#handleClientOutputBufferLimitForSubscribeTooSlow, redis.clients.jedis.tests.commands.PublishSubscribeCommandsTest#binaryPsubscribeMany, redis.clients.jedis.tests.commands.PublishSubscribeCommandsTest#psubscribe, redis.clients.jedis.tests.commands.PublishSubscribeCommandsTest#pubSubNumSub, redis.clients.jedis.tests.commands.PublishSubscribeCommandsTest#subscribeLazily, redis.clients.jedis.tests.commands.PublishSubscribeCommandsTest#pubSubChannels, redis.clients.jedis.tests.commands.PublishSubscribeCommandsTest#binarySubscribeMany, redis.clients.jedis.tests.commands.PublishSubscribeCommandsTest#pubSubNumPat, redis.clients.jedis.tests.commands.PublishSubscribeCommandsTest#binarySubscribe, redis.clients.jedis.tests.commands.PublishSubscribeCommandsTest#binaryPsubscribe, redis.clients.jedis.tests.commands.PublishSubscribeCommandsTest#subscribeMany, redis.clients.jedis.tests.commands.PublishSubscribeCommandsTest#psubscribeMany, redis.clients.jedis.tests.commands.PublishSubscribeCommandsTest#unsubscribeWhenNotSusbscribed, redis.clients.jedis.tests.commands.PublishSubscribeCommandsTest#binarySubscribeLazily, redis.clients.jedis.tests.commands.PublishSubscribeCommandsTest#subscribe, redis.clients.jedis.tests.commands.PublishSubscribeCommandsTest#pubSubChannelWithPingPong], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=16}
status: NOPATCH
dateEnd: Mon Oct 23 20:14:24 CEST 2017
allocatedtime: 62minutes 
passingTime: 5minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/291610330/src/main/java], projectClasspath='[Ljava.net.URL;@74ecc76c', projectTests=[redis.clients.jedis.tests.commands.PublishSubscribeCommandsTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 32
nbAngelicValues: 26
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #17
location: FailureLocation{className='redis.clients.jedis.tests.JedisTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.JedisTest#timeoutConnectionWithURI, redis.clients.jedis.tests.JedisTest#startWithUrl, redis.clients.jedis.tests.JedisTest#shouldReconnectToSameDB, redis.clients.jedis.tests.JedisTest#timeoutConnection, redis.clients.jedis.tests.JedisTest#checkCloseable, redis.clients.jedis.tests.JedisTest#checkBinaryData, redis.clients.jedis.tests.JedisTest#checkDisconnectOnQuit, redis.clients.jedis.tests.JedisTest#startWithUrlString, redis.clients.jedis.tests.JedisTest#failWhenSendingNullValues, redis.clients.jedis.tests.JedisTest#useWithoutConnecting, redis.clients.jedis.tests.JedisTest#shouldNotUpdateDbIndexIfSelectFails, redis.clients.jedis.tests.JedisTest#allowUrlWithNoDBAndNoPassword, redis.clients.jedis.tests.JedisTest#connectWithShardInfo, redis.clients.jedis.tests.JedisTest#shouldThrowInvalidURIExceptionForInvalidURI], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=14}
status: NOPATCH
dateEnd: Mon Oct 23 20:19:44 CEST 2017
allocatedtime: 60minutes 
passingTime: 5minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/291610330/src/main/java], projectClasspath='[Ljava.net.URL;@6f89e48', projectTests=[redis.clients.jedis.tests.JedisTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 32
nbAngelicValues: 26
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #18
location: FailureLocation{className='redis.clients.jedis.tests.ShardedJedisPoolTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.ShardedJedisPoolTest#checkFailedJedisServer, redis.clients.jedis.tests.ShardedJedisPoolTest#checkPoolRepairedWhenJedisIsBroken, redis.clients.jedis.tests.ShardedJedisPoolTest#checkJedisIsReusedWhenReturned, redis.clients.jedis.tests.ShardedJedisPoolTest#checkResourceIsCloseable, redis.clients.jedis.tests.ShardedJedisPoolTest#startWithUrlString, redis.clients.jedis.tests.ShardedJedisPoolTest#checkCloseableConnections, redis.clients.jedis.tests.ShardedJedisPoolTest#checkConnections, redis.clients.jedis.tests.ShardedJedisPoolTest#shouldNotShareInstances, redis.clients.jedis.tests.ShardedJedisPoolTest#returnResourceShouldResetState, redis.clients.jedis.tests.ShardedJedisPoolTest#checkPoolOverflow, redis.clients.jedis.tests.ShardedJedisPoolTest#startWithUrl, redis.clients.jedis.tests.ShardedJedisPoolTest#shouldReturnActiveShardsWhenOneGoesOffline, redis.clients.jedis.tests.ShardedJedisPoolTest#checkConnectionWithDefaultPort], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=13}
status: NOPATCH
dateEnd: Mon Oct 23 20:27:32 CEST 2017
allocatedtime: 57minutes 
passingTime: 7minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/291610330/src/main/java], projectClasspath='[Ljava.net.URL;@715d9d0f', projectTests=[redis.clients.jedis.tests.ShardedJedisPoolTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 55
nbAngelicValues: 35
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #19
location: FailureLocation{className='redis.clients.jedis.tests.commands.ClusterBinaryJedisCommandsTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.commands.ClusterBinaryJedisCommandsTest#testGetSlot, redis.clients.jedis.tests.commands.ClusterBinaryJedisCommandsTest#testHmset, redis.clients.jedis.tests.commands.ClusterBinaryJedisCommandsTest#testRpush, redis.clients.jedis.tests.commands.ClusterBinaryJedisCommandsTest#testIncr, redis.clients.jedis.tests.commands.ClusterBinaryJedisCommandsTest#redis.clients.jedis.tests.commands.ClusterBinaryJedisCommandsTest, redis.clients.jedis.tests.commands.ClusterBinaryJedisCommandsTest#testBinaryGetAndSet, redis.clients.jedis.tests.commands.ClusterBinaryJedisCommandsTest#testSadd], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='java.lang.NullPointerException', failureDetail='null', isError=true}], nbFailures=0, nbErrors=13}
status: NOPATCH
dateEnd: Mon Oct 23 20:36:22 CEST 2017
allocatedtime: 54minutes 
passingTime: 8minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/291610330/src/main/java], projectClasspath='[Ljava.net.URL;@5caf90c4', projectTests=[redis.clients.jedis.tests.commands.ClusterBinaryJedisCommandsTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 62
nbAngelicValues: 39
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #20
location: FailureLocation{className='redis.clients.jedis.tests.commands.ControlCommandsTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.commands.ControlCommandsTest#save, redis.clients.jedis.tests.commands.ControlCommandsTest#lastsave, redis.clients.jedis.tests.commands.ControlCommandsTest#bgrewriteaof, redis.clients.jedis.tests.commands.ControlCommandsTest#sync, redis.clients.jedis.tests.commands.ControlCommandsTest#info, redis.clients.jedis.tests.commands.ControlCommandsTest#configSet, redis.clients.jedis.tests.commands.ControlCommandsTest#debug, redis.clients.jedis.tests.commands.ControlCommandsTest#monitor, redis.clients.jedis.tests.commands.ControlCommandsTest#configGet, redis.clients.jedis.tests.commands.ControlCommandsTest#waitReplicas, redis.clients.jedis.tests.commands.ControlCommandsTest#bgsave, redis.clients.jedis.tests.commands.ControlCommandsTest#readonly], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=12}
status: NOPATCH
dateEnd: Mon Oct 23 20:41:36 CEST 2017
allocatedtime: 50minutes 
passingTime: 5minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/291610330/src/main/java], projectClasspath='[Ljava.net.URL;@269ee42b', projectTests=[redis.clients.jedis.tests.commands.ControlCommandsTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 32
nbAngelicValues: 26
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #21
location: FailureLocation{className='redis.clients.jedis.tests.commands.BitCommandsTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.commands.BitCommandsTest#testBitfield, redis.clients.jedis.tests.commands.BitCommandsTest#bitpos, redis.clients.jedis.tests.commands.BitCommandsTest#bitposBinary, redis.clients.jedis.tests.commands.BitCommandsTest#bitOp, redis.clients.jedis.tests.commands.BitCommandsTest#bitOpNot, redis.clients.jedis.tests.commands.BitCommandsTest#setAndgetbit, redis.clients.jedis.tests.commands.BitCommandsTest#testBinaryBitfield, redis.clients.jedis.tests.commands.BitCommandsTest#bitCount, redis.clients.jedis.tests.commands.BitCommandsTest#bitposWithNoMatchingBitExist, redis.clients.jedis.tests.commands.BitCommandsTest#setAndgetrange, redis.clients.jedis.tests.commands.BitCommandsTest#bitposWithNoMatchingBitExistWithinRange], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=11}
status: NOPATCH
dateEnd: Mon Oct 23 20:46:33 CEST 2017
allocatedtime: 47minutes 
passingTime: 4minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/291610330/src/main/java], projectClasspath='[Ljava.net.URL;@4de2c8e8', projectTests=[redis.clients.jedis.tests.commands.BitCommandsTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 32
nbAngelicValues: 26
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #22
location: FailureLocation{className='redis.clients.jedis.tests.JedisSentinelTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.JedisSentinelTest#sentinelFailover, redis.clients.jedis.tests.JedisSentinelTest#sentinelSet, redis.clients.jedis.tests.JedisSentinelTest#sentinelRemove, redis.clients.jedis.tests.JedisSentinelTest#sentinel, redis.clients.jedis.tests.JedisSentinelTest#sentinelMonitor], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:26379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:26379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:26381', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:26379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:26379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:26379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:26379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:26379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:26379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:26379', isError=true}], nbFailures=0, nbErrors=10}
status: NOPATCH
dateEnd: Mon Oct 23 20:53:41 CEST 2017
allocatedtime: 45minutes 
passingTime: 7minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/291610330/src/main/java], projectClasspath='[Ljava.net.URL;@65426b2f', projectTests=[redis.clients.jedis.tests.JedisSentinelTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 54
nbAngelicValues: 38
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #23
location: FailureLocation{className='redis.clients.jedis.tests.SSLJedisTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.SSLJedisTest#connectWithoutShardInfo, redis.clients.jedis.tests.SSLJedisTest#connectWithShardInfoAndEmptyTrustStore, redis.clients.jedis.tests.SSLJedisTest#connectWithShardInfoAndCustomHostnameVerifier, redis.clients.jedis.tests.SSLJedisTest#connectWithShardInfoAndCustomSocketFactory, redis.clients.jedis.tests.SSLJedisTest#connectWithShardInfoAndCustomHostnameVerifierByIpAddress, redis.clients.jedis.tests.SSLJedisTest#connectWithShardInfo, redis.clients.jedis.tests.SSLJedisTest#connectWithUrl, redis.clients.jedis.tests.SSLJedisTest#connectWithShardInfoByIpAddress], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=8}
status: NOPATCH
dateEnd: Mon Oct 23 20:54:12 CEST 2017
allocatedtime: 42minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/291610330/src/main/java], projectClasspath='[Ljava.net.URL;@7408fd47', projectTests=[redis.clients.jedis.tests.SSLJedisTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 2
nbAngelicValues: 2
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #24
location: FailureLocation{className='redis.clients.jedis.tests.commands.GeoCommandsTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.commands.GeoCommandsTest#geodist, redis.clients.jedis.tests.commands.GeoCommandsTest#geoadd, redis.clients.jedis.tests.commands.GeoCommandsTest#georadiusByMember, redis.clients.jedis.tests.commands.GeoCommandsTest#geopos, redis.clients.jedis.tests.commands.GeoCommandsTest#georadiusByMemberBinary, redis.clients.jedis.tests.commands.GeoCommandsTest#georadius, redis.clients.jedis.tests.commands.GeoCommandsTest#geohash, redis.clients.jedis.tests.commands.GeoCommandsTest#georadiusBinary], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=8}
status: NOPATCH
dateEnd: Mon Oct 23 20:59:03 CEST 2017
allocatedtime: 42minutes 
passingTime: 4minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/291610330/src/main/java], projectClasspath='[Ljava.net.URL;@311bb654', projectTests=[redis.clients.jedis.tests.commands.GeoCommandsTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 32
nbAngelicValues: 26
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #25
location: FailureLocation{className='redis.clients.jedis.tests.JedisSentinelPoolTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.JedisSentinelPoolTest#customClientName, redis.clients.jedis.tests.JedisSentinelPoolTest#checkResourceIsCloseable, redis.clients.jedis.tests.JedisSentinelPoolTest#testResetInvalidPassword, redis.clients.jedis.tests.JedisSentinelPoolTest#returnResourceShouldResetState, redis.clients.jedis.tests.JedisSentinelPoolTest#checkCloseableConnections, redis.clients.jedis.tests.JedisSentinelPoolTest#ensureSafeTwiceFailover, redis.clients.jedis.tests.JedisSentinelPoolTest#testResetValidPassword], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='All sentinels down, cannot determine where is mymaster master is running...', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='All sentinels down, cannot determine where is mymaster master is running...', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='All sentinels down, cannot determine where is mymaster master is running...', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='All sentinels down, cannot determine where is mymaster master is running...', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='All sentinels down, cannot determine where is mymaster master is running...', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='All sentinels down, cannot determine where is mymaster master is running...', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='All sentinels down, cannot determine where is mymaster master is running...', isError=true}], nbFailures=0, nbErrors=7}
status: NOPATCH
dateEnd: Mon Oct 23 21:12:55 CEST 2017
allocatedtime: 40minutes 
passingTime: 13minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/291610330/src/main/java], projectClasspath='[Ljava.net.URL;@7fa8614f', projectTests=[redis.clients.jedis.tests.JedisSentinelPoolTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 110
nbAngelicValues: 69
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #26
location: FailureLocation{className='redis.clients.jedis.tests.commands.HyperLogLogCommandsTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.commands.HyperLogLogCommandsTest#pfmerge, redis.clients.jedis.tests.commands.HyperLogLogCommandsTest#pfmergeBinary, redis.clients.jedis.tests.commands.HyperLogLogCommandsTest#pfadd, redis.clients.jedis.tests.commands.HyperLogLogCommandsTest#pfaddBinary, redis.clients.jedis.tests.commands.HyperLogLogCommandsTest#pfcounts, redis.clients.jedis.tests.commands.HyperLogLogCommandsTest#pfcount, redis.clients.jedis.tests.commands.HyperLogLogCommandsTest#pfcountBinary], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=7}
status: NOPATCH
dateEnd: Mon Oct 23 21:17:59 CEST 2017
allocatedtime: 33minutes 
passingTime: 5minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/291610330/src/main/java], projectClasspath='[Ljava.net.URL;@2e4e1cef', projectTests=[redis.clients.jedis.tests.commands.HyperLogLogCommandsTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 32
nbAngelicValues: 26
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #27
location: FailureLocation{className='redis.clients.jedis.tests.commands.SortingCommandsTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.commands.SortingCommandsTest#sort, redis.clients.jedis.tests.commands.SortingCommandsTest#sortAlpha, redis.clients.jedis.tests.commands.SortingCommandsTest#sortLimit, redis.clients.jedis.tests.commands.SortingCommandsTest#sortBy, redis.clients.jedis.tests.commands.SortingCommandsTest#sortStore, redis.clients.jedis.tests.commands.SortingCommandsTest#sortGet, redis.clients.jedis.tests.commands.SortingCommandsTest#sortDesc], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=7}
status: NOPATCH
dateEnd: Mon Oct 23 21:22:50 CEST 2017
allocatedtime: 31minutes 
passingTime: 4minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/291610330/src/main/java], projectClasspath='[Ljava.net.URL;@c7e5e75', projectTests=[redis.clients.jedis.tests.commands.SortingCommandsTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 32
nbAngelicValues: 26
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #28
location: FailureLocation{className='redis.clients.jedis.tests.commands.VariadicCommandsTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.commands.VariadicCommandsTest#zrem, redis.clients.jedis.tests.commands.VariadicCommandsTest#hdel, redis.clients.jedis.tests.commands.VariadicCommandsTest#sadd, redis.clients.jedis.tests.commands.VariadicCommandsTest#rpush, redis.clients.jedis.tests.commands.VariadicCommandsTest#zadd, redis.clients.jedis.tests.commands.VariadicCommandsTest#lpush], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=6}
status: NOPATCH
dateEnd: Mon Oct 23 21:27:41 CEST 2017
allocatedtime: 29minutes 
passingTime: 4minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/291610330/src/main/java], projectClasspath='[Ljava.net.URL;@6695b98e', projectTests=[redis.clients.jedis.tests.commands.VariadicCommandsTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 32
nbAngelicValues: 26
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #29
location: FailureLocation{className='redis.clients.jedis.tests.ShardedJedisPipelineTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.ShardedJedisPipelineTest#pipelineResponse, redis.clients.jedis.tests.ShardedJedisPipelineTest#testSyncWithNoCommandQueued, redis.clients.jedis.tests.ShardedJedisPipelineTest#pipeline, redis.clients.jedis.tests.ShardedJedisPipelineTest#pipelineResponseWithinPipeline, redis.clients.jedis.tests.ShardedJedisPipelineTest#canRetrieveUnsetKey], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=5}
status: NOPATCH
dateEnd: Mon Oct 23 21:33:43 CEST 2017
allocatedtime: 27minutes 
passingTime: 6minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/291610330/src/main/java], projectClasspath='[Ljava.net.URL;@3f0cd524', projectTests=[redis.clients.jedis.tests.ShardedJedisPipelineTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 45
nbAngelicValues: 30
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #30
location: FailureLocation{className='redis.clients.jedis.tests.ShardedJedisTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.ShardedJedisTest#tryShardingWithMurmure, redis.clients.jedis.tests.ShardedJedisTest#testAvoidLeaksUponDisconnect, redis.clients.jedis.tests.ShardedJedisTest#checkCloseable, redis.clients.jedis.tests.ShardedJedisTest#trySharding], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6380', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6380', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6380', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=4}
status: NOPATCH
dateEnd: Mon Oct 23 21:51:19 CEST 2017
allocatedtime: 24minutes 
passingTime: 17minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/291610330/src/main/java], projectClasspath='[Ljava.net.URL;@7054dede', projectTests=[redis.clients.jedis.tests.ShardedJedisTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 149
nbAngelicValues: 86
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #31
location: FailureLocation{className='redis.clients.jedis.tests.commands.ObjectCommandsTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.commands.ObjectCommandsTest#objectEncoding, redis.clients.jedis.tests.commands.ObjectCommandsTest#objectIdletime, redis.clients.jedis.tests.commands.ObjectCommandsTest#objectRefcount], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=3}
status: NOPATCH
dateEnd: Mon Oct 23 21:56:29 CEST 2017
allocatedtime: 15minutes 
passingTime: 5minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/291610330/src/main/java], projectClasspath='[Ljava.net.URL;@2878f2', projectTests=[redis.clients.jedis.tests.commands.ObjectCommandsTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 35
nbAngelicValues: 28
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #32
location: FailureLocation{className='redis.clients.jedis.tests.commands.ConnectionHandlingCommandsTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.commands.ConnectionHandlingCommandsTest#quit, redis.clients.jedis.tests.commands.ConnectionHandlingCommandsTest#binary_quit], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=2}
status: NOPATCH
dateEnd: Mon Oct 23 22:01:31 CEST 2017
allocatedtime: 13minutes 
passingTime: 5minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/291610330/src/main/java], projectClasspath='[Ljava.net.URL;@413d8420', projectTests=[redis.clients.jedis.tests.commands.ConnectionHandlingCommandsTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 32
nbAngelicValues: 26
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #33
location: FailureLocation{className='redis.clients.jedis.tests.TupleSortedSetTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.TupleSortedSetTest#testString, redis.clients.jedis.tests.TupleSortedSetTest#testBinary], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=2}
status: NOPATCH
dateEnd: Mon Oct 23 22:06:23 CEST 2017
allocatedtime: 10minutes 
passingTime: 4minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/291610330/src/main/java], projectClasspath='[Ljava.net.URL;@4a9e3e8e', projectTests=[redis.clients.jedis.tests.TupleSortedSetTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 32
nbAngelicValues: 26
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #34
location: FailureLocation{className='redis.clients.jedis.tests.ConnectionCloseTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.ConnectionCloseTest#connectIfNotConnectedWhenSettingTimeoutInfinite], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=1}
status: NOPATCH
dateEnd: Mon Oct 23 22:09:56 CEST 2017
allocatedtime: 8minutes 
passingTime: 3minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/291610330/src/main/java], projectClasspath='[Ljava.net.URL;@5b9957a8', projectTests=[redis.clients.jedis.tests.ConnectionCloseTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 19
nbAngelicValues: 18
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #35
location: FailureLocation{className='redis.clients.jedis.tests.commands.SlowlogCommandsTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.commands.SlowlogCommandsTest#slowlog], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=1}
status: TIMEOUT
dateEnd: Mon Oct 23 22:26:10 CEST 2017
allocatedtime: 7minutes 
passingTime: 16minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/291610330/src/main/java], projectClasspath='[Ljava.net.URL;@76e0bc4e', projectTests=[redis.clients.jedis.tests.commands.SlowlogCommandsTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 32
nbAngelicValues: 1
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #36
location: FailureLocation{className='redis.clients.jedis.tests.ModuleTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.ModuleTest#testModules], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=1}
status: TIMEOUT
dateEnd: Mon Oct 23 22:28:52 CEST 2017
allocatedtime: 2minutes 
passingTime: 2minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/291610330/src/main/java], projectClasspath='[Ljava.net.URL;@402563cf', projectTests=[redis.clients.jedis.tests.ModuleTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 32
nbAngelicValues: 2
ignoreStatus: NOTHING_TO_IGNORE
----------



